_episodes/03-data-structures.md

```diff
@@ -39,6 +57,16 @@ keypoints:
 >
 > > ## Answer
 > >
-> > A.  `country`, because it is a 'categorical' variable
+> > A.  `country`, because it is a __categorical__ variable
 > {: .solution}
 {: .challenge}
+
+## Tidy data follows a set of rules.
+
+These rules keep data well organized and ready for analysis.
+
+1. Order doesn’t matter
+2. No duplicate rows
+3. Every cell contains one value
+4. One column per type of information
+5. No redundant information
```

Comment:
Include link to Wickham's "Tidy Data" paper `http://vita.had.co.nz/papers/tidy-data.pdf`